gnd/src/main/java/com/google/android/gnd/rx/Action.java

```diff
@@ -28,19 +28,15 @@ protected Action() {
     this.handled = false;
   }
 
-  /**
-   * Invokes the provided handler if the value has not yet been handled.
-   *
-   * @param handler
-   */
+  public static Action create() {
+    return new Action();
+  }
+
+  /** Invokes the provided handler if the value has not yet been handled. */
```

Comment:
Just an observation. It feels like the diff got confused with the moving of `create` to above `ifUnhandled`. Might be better to keep the ordering the same to avoid "noise" in future PRs.